docs/tutorial.rst

```diff
@@ -115,6 +114,10 @@ Let us fetch our new collection of tasks:
     }
 
 
+Keep a note of the ``ETag`` and of the ``last_modified`` values returned (here
+respectively ``"1434641474977"`` and ``"1434641515332"``), we'll need them for
```

Comment:
Yes this was fixed in Kinto 1.2, it is related to this bug: https://github.com/mozilla-services/kinto/pull/80